src/ref/ui-patterns/mobile/carousel-mobile.md

```diff
@@ -0,0 +1,169 @@
+---
+summary: 
+tags: runtime-mobileandreactiveweb
+---
+
+# Carousel UI Pattern Reference
+
+### Customizing the dots
```

Comment:
Jumped from heading 1 (line 6) to heading 3. Please change these heading 3 titles to heading 2. Also check remaining headers in the file.